pkg/tekton/pipeline_info.go

```diff
@@ -193,14 +195,13 @@ func CreatePipelineRunInfo(prName string, podList *corev1.PodList, ps *v1.Pipeli
 					gitURL = value
 				case "-revision":
 					if shaRegexp.MatchString(value) {
-						lastCommitSha = value
+						shaFromGitInit = value
 					} else {
 						branch = value
 					}
 				}
 			}
 		}
-		var pullPullSha string
```

Comment:
So you are just moving this up to the rest of the var declaration, right

Reply:
Yup. I couldn't see a reason to handle `pullPullSha` differently than `pullBaseSha` in this respect - they're both pulling from env vars set by Prow/Lighthouse that'll be on all the containers, so why are we putting one in a different scope than the other?